src/stylesheets/_variables.scss

```diff
@@ -61,16 +95,16 @@ $color-background-success: $color-green !default;
 
 $color-border: $color-grey-light !default;
 $color-border-light: $color-grey-lighter !default;
-$color-border-active: $color-blue !default;
+$color-border-active: $um-blue-5 !default;
```

Comment:
We want to move away from this. These variables that refer another `$color-*` variable are used throughout the code base but we need to selectively go through the code base and replace these with the `$um-*` variables where applicable.

I would start by making the button have the same look and feel as the [button](/Ultimaker/stardust-web/blob/master/src/components/buttons/_um-button.scss) in `stardust-web`. Other components can follow in separate smaller pull requests. Coordinate with design to make sure everything looks the way it should.

Reply:
I think it would be best if you added what needs to be done to the ticket:
https://jira.ultimaker.com/browse/CS-781

It's not mentioned that we want to "refarctor" the CSS as part of this story.

Reply:
It was discussed (and probably cut a bit short) during refinement. I'll add it.

Reply:
Thanks!